Dota 2 Reborn Keyboard Setups/Loopuleasa's Super Compact QWERTG-DFXCV layout (ALT,SPACE mods)/dota2_gameplay_mode/dota2_keybinds_default.cfg

```diff
@@ -63,17 +63,21 @@ bind "Z" "dota_recent_event;"
 bind "I" "inspectheroinworld"
 
 //  The micromanagement section
-bind "2" "+dota_control_group 1"
-bind "3" "+dota_control_group 2"
-bind "4" "+dota_control_group 3"
-bind "5" "+dota_control_group 4"
-bind "6" "+dota_control_group 5"
-bind "7" "+dota_control_group 6"
-bind "8" "+dota_control_group 7"
-bind "9" "+dota_control_group 8"
-bind "0" "+dota_control_group 9"
+bind "2" "+two"
+bind "3" "+three"
+bind "4" "+four"
+bind "5" "+five"
+bind "6" "+six"
+bind "7" "+seven"
+bind "8" "+eight"
+bind "9" "+nine"
+bind "0" "+ten"
```

Comment:
I'd suggest using a different name for that. Like `quick_control_group_10` or something smiliar, more descriptive.